build/build-image/Dockerfile

```diff
@@ -23,7 +23,7 @@ RUN apt-get update && \
 
 # install go
 WORKDIR /usr/local
-ENV GO_VERSION=1.14.12
+ENV GO_VERSION=1.15.13
 ENV GOPATH /go
 ENV GO111MODULE=on
```

Comment:
It looks like we will be able to get rid of this env var with go 1.16 (where on becomes the default).